lib/maths/CGammaRateConjugate.cc

```diff
@@ -691,6 +691,17 @@ class CLogMarginalLikelihood : core::CNonCopyable {
 
 } // detail::
 
+const std::string READABLE_OFFSET_TAG("offset");
```

Comment:
It might be less error prone to define the readable and minified tags on the same line.  This could be something like:

```
const TStrStrPr OFFSET_TAG("a", "offset");
```

(where `TStrStrPr` is a `std::pair<std::string, std::string>`)

Then overload `insertValue` so that the first argument can be either a `std::string` or `TStrStrPr`.  This would avoid the need to have all the `readableTags ? READABLE_FOO_TAG : FOO_TAG` constructs as the test for readable tags would move into the `TStrStrPr` overload of `insertValue`.

Alternatively a custom class could be used:

```
const core::CPersistTag OFFSET_TAG("a", "offset");
```

To start with it would basically just be a pair of strings, but would be more future proof if we ever wanted to add another way of creating tags in the future, as we wouldn't need to search and replace a load of `TStrStrPr`s with `core::CPersistTag` at that future time.